programmer/initialize/initialize8MB.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh -e
```

Comment:
This file is specific to the ULX3S board, it doesn't belong in a generic path.  Instead it should go to the board-specific location of the repository, something like: `boards/ULX3S/initialize`